tests/airlock.test.js

```diff
@@ -1,15 +1,75 @@
 const axios = require("axios");
+const utils = require("./utils/utils");
 
-const { SERVER_URL = "http://localhost:8000/api" } = process.env;
+const {
+    SERVER_URL = "http://localhost:8000/api",
+    DEFAULT_INVALID_TEST_TOKEN = "INVALID_TOKEN"
+} = process.env;
 
 describe("Given Airlock is running", () => {
+    let DEFAULT_TEST_TOKEN;
+
+    beforeAll(async () => {
+        DEFAULT_TEST_TOKEN = await utils.generateTestToken();
+    });
+
+    describe("When I try to access to the mock service with no token", () => {
+        it("Then replies with a status code 401", async () => {
+            try {
+                await axios.get(`${SERVER_URL}/auth`);
+
+                throw new Error(
+                    "Should fail with a 401 error for client request errors"
+                );
+            } catch (err) {
+                expect(err.response.status).toEqual(401);
+            }
+        });
+    });
+
+    describe("When I try to access to the mock service with a valid token", () => {
+        it("Then replies with a status code 200", async () => {
+            let req;
+
+            req = await axios.get(`${SERVER_URL}/auth`, {
+                headers: {
+                    Authorization: ["Bearer", DEFAULT_TEST_TOKEN].join(" ")
+                }
+            });
+
+            expect(req.status).toEqual(200);
+        });
+    });
+
+    describe("When I try to access to the mock service with an invalid token", () => {
+        it("Then replies with a status code 403", async () => {
+            try {
+                await axios.get(`${SERVER_URL}/auth`, {
+                    headers: {
+                        Authorization: ["Bearer", DEFAULT_INVALID_TEST_TOKEN].join(" ")
+                    }
+                });
+
+                throw new Error(
+                    "Should fail with a 403 error for client request errors"
+                );
+            } catch (err) {
+                expect(err.response.status).toEqual(403);
+            }
+        });
+    });
+
     describe("When I add an item to the mock service using a POST request", () => {
         let data;
 
         beforeAll(async () => {
             const response = await axios.post(`${SERVER_URL}/item`, {
                 id: 1,
                 name: "KB9"
+            }, {
+                headers: {
+                    Authorization: ["Bearer", DEFAULT_TEST_TOKEN].join(" ")
```

Comment:
let's use a template literal for this `Bearer ${ DEFAULT_TEST_TOKEN}`

Reply:
Indeed, I originally did an interceptor but choose to delete it as I thought it would be out of scope.